src/EditorFeatures/CSharpTest/Microsoft.CodeAnalysis.CSharp.EditorFeatures.UnitTests.csproj

```diff
@@ -27,11 +27,6 @@
     <ProjectReference Include="..\..\Compilers\Test\Resources\Core\Microsoft.CodeAnalysis.Compiler.Test.Resources.csproj" />
     <ProjectReference Include="..\DiagnosticsTestUtilities\Microsoft.CodeAnalysis.EditorFeatures.DiagnosticsTests.Utilities.csproj" />
     <ProjectReference Include="..\TestUtilities\Microsoft.CodeAnalysis.EditorFeatures.Test.Utilities.csproj" />
-    <ProjectReference Include="..\..\Compilers\VisualBasic\Portable\Microsoft.CodeAnalysis.VisualBasic.vbproj" />
-    <ProjectReference Include="..\VisualBasic\Microsoft.CodeAnalysis.VisualBasic.EditorFeatures.vbproj" />
-    <ProjectReference Include="..\..\Features\VisualBasic\Portable\Microsoft.CodeAnalysis.VisualBasic.Features.vbproj" />
-    <ProjectReference Include="..\..\Scripting\VisualBasic\Microsoft.CodeAnalysis.VisualBasic.Scripting.vbproj" />
```

Comment:
I actually noticed that this is the only reference that is no longer being built as part of this change. So other references will still be existing as transitive references. Should I only remove this one? or it's okay to remove all?

Reply:
I don't have strong opinions on whether we should or should not explicitly include transitive references anymore. Historically I was strongly for it but that was for technical reasons. There was a lot of tooling that simply failed to function properly if we didn't have them. That is why I pushed for them to be included and added verification that we did so. Now though the bugs are gone and there is no longer a technical need to do this. 

My only issue though is that people understand what the change is doing. For instance, as you noted, this change in no way removes the dependency on the VB libraries from this project. As long as this project continues to reference the core compilers testing project the dependency remains, it's just hidden.

Reply:
There is a *single* reference that is completely removed. I can update the PR to only remove this one. But as those technical reasons gone and those bugs are fixed. I personally lean towards removing all that are not needed.

While the removal feels redundant (except for this single reference), it may be beneficial in case whatever project causing the transitive dependency didn't do that any more in future.

But certainly you'd know better than me in this area. So I'm open to both options. Either remove only the single reference that isn't a transitive dependency or remove all unneeded references.